sklearn/utils/tests/conftest.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+
+import sklearn
+
+
+@pytest.fixture(scope='function')
+def print_changed_only_setter(value):
```

Comment:
The fixture isn't strictly needed but it makes tests a bit easier

Reply:
I would rather like a feature named `repr_all_params` that does not require use to repeat `pytest.mark.parametrize('value', [False])` each time.